static/js/components/ExecutedObservationsTable.jsx

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Paper from "@material-ui/core/Paper";
+import {
+  makeStyles,
+  createTheme,
+  MuiThemeProvider,
+  useTheme,
+} from "@material-ui/core/styles";
+import CircularProgress from "@material-ui/core/CircularProgress";
+
+// eslint-disable-next-line
```

Comment:
Just curious: what did lint not like about this?

Reply:
Yeah clearly can be removed!